pkgs/tools/misc/desktop-file-utils/default.nix

```diff
@@ -4,7 +4,7 @@ with lib;
 
 stdenv.mkDerivation rec {
   pname = "desktop-file-utils";
-  version = "0.24";
+  version = "0.26";
 
   src = fetchurl {
     url = "https://www.freedesktop.org/software/${pname}/releases/${pname}-${version}.tar.xz";
```

Comment:
Missing hash update. Also this seems unrelated to this PR.

Reply:
Ok, let's reduce the scope to get this merged faster